karaf/pom.xml

```diff
@@ -0,0 +1,138 @@
+    THE SOFTWARE.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>influxdb-client</artifactId>
+        <groupId>com.influxdb</groupId>
+        <version>2.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>influxdb-karaf</artifactId>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>karaf-features</module>
+        <module>karaf-kar</module>
+        <module>karaf-assembly</module>
+    </modules>
+
+    <name>Apache Karaf Integration for InfluxDB 2.0</name>
+    <description>
+        Apache Karaf Integration for InfluxDB 2.0: features and KAR file.
+    </description>
+
+    <url>https://github.com/influxdata/influxdb-client-java/tree/master/karaf</url>
+
+    <licenses>
+        <license>
+            <name>The MIT License (MIT)</name>
+            <url>https://raw.githubusercontent.com/influxdata/influxdb-client-java/master/LICENSE</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>József Börcsök</name>
+            <email>borcsokj@gmail.com</email>
+            <url>http://www.github.com/borcsokj</url>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:git@github.com:influxdata/influxdb-client-java.git</connection>
+        <developerConnection>scm:git:git@github.com:influxdata/influxdb-client-java.git</developerConnection>
+        <url>http://github.com/influxdata/influxdb-client-java/tree/master</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <properties>
+        <karaf.version>4.2.11</karaf.version>
+    </properties>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.karaf.tooling</groupId>
+                    <artifactId>karaf-maven-plugin</artifactId>
+                    <version>${karaf.version}</version>
+                    <extensions>true</extensions>
+                    <configuration>
+                        <enableGeneration>true</enableGeneration>
+                        <aggregateFeatures>true</aggregateFeatures>
+                    </configuration>
+                </plugin>
+
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-install-plugin</artifactId>
+                    <version>2.5.2</version>
+                </plugin>
+
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-deploy-plugin</artifactId>
+                    <version>2.8.2</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
```

Comment:
Please, move `pluginManagement` into parent `pom.xml`. It will help us to avoid using multiple version of plugins.